lms/djangoapps/teams/static/teams/js/spec/views/topics_spec.js

```diff
@@ -33,12 +33,14 @@ define([
                     "num_pages": 2,
```

Comment:
Should this be removed too? Given that there are two tests that do this, I'd like to see the creation of a mock topic collection moved into TeamSpecHelpers.